src/Storefront/Resources/app/storefront/src/plugin/main-menu/offcanvas-menu.plugin.js

```diff
@@ -13,7 +13,9 @@ export default class OffcanvasMenuPlugin extends Plugin {
     static options = {
         navigationUrl: window.router['frontend.menu.offcanvas'],
         position: 'left',
+        /** @deprecated tag:v6.7.0 - property "tiggerEvent" will be removed. Use "triggerEvent" instead */
         tiggerEvent: 'click',
```

Comment:
But `Tigger` 😢 
![](https://i.pinimg.com/originals/4a/05/cf/4a05cf18a6b71d69efa3847205b42c35.jpg)

Reply:
exactly what I was thinking off, when I read this :rofl: